crates/iota-core/src/authority/scorer.rs

```diff
@@ -30,45 +43,67 @@ impl Scorer {
         let committee_size = voting_power.len();
         match protocol_config.scorer_version_as_option() {
             None | Some(1) => {
+                // Local metrics count are always initialized as zero.
                 let current_local_metrics_count = Arc::new(VersionedScoringMetrics::new(
                     committee_size,
                     protocol_config,
                 ));
 
-                let (received_metrics, metrics_missing_from, current_scores, invalid_reports_count) =
+                let max_score = 2_u64.pow(16);
+                let (received_metrics, has_not_sent_report, current_scores, invalid_reports_count) =
                     (0..committee_size)
                         .map(|_| {
                             (
+                                // Received metrics initialized to zero.
                                 VersionedScoringMetrics::new(committee_size, protocol_config),
+                                // Initially, none of the authorities had sent any valid report.
                                 AtomicBool::new(true),
-                                AtomicU64::new(0),
+                                // Current scores initialized to max score.
+                                AtomicU64::new(max_score),
+                                // Invalid reports count initialized to zero.
                                 AtomicU64::new(0),
                             )
                         })
                         .collect();
+                let parameters = ParametersV1 {
+                    max_score,
```

Comment:
Can we just use `max_score` here in place of `2_u64.pow(16)` in all these params? If we changed `max_score` would these all change too? Or is it a separate value?